File: dosya_11/list.py
input_list = ["None","None","None"]


ad = input("Ad: ")
soyad = input("Soyad: ")
yaş = input("Yaş: ")

input_list[0] = ad
input_list[1] = soyad
input_list[2] = yaş

def yazdir(liste):

    try:
        liste = list(liste)
    except:
        print("HATAAAA:::!:!:!:!:")

    print(f"""

    Kullanıcının:

    Adı: {liste[0]}

    Soyadı: {liste[1]}

    Yaşı: {liste[2]}

    """)

yazdir(input_list)

File: dosya_11/list.js
const prompt = require("prompt-sync")({ sigint: true });


let input_list = []

ad = prompt("Adın ne? ")

soyad = prompt("Soyadını yuaz.. : ")

yas = prompt("Bu geçici dünyaya geldiğin yılı yazınız: ")
  
function yazdir (liste){
    text = `Kullanıcının:
    Adı: ${ad}\n
    Soyadı: ${soyad}\n
    Yaşı: ${yas}`
    console.log(text)
}

yazdir();





  



